File: pl/fortran/hw1.f90
! author y.emre can ocak 9119001276
! platform Linux 5.4.18-1-MANJARO
! compiled with gcc-fortran 9.2 f90 ext. 

program  minMaxArg
   implicit  none
   integer :: minimum, maximum, noi           
   real :: sum
   integer :: count                           
   integer :: input                           
   noi = 8
   do  count = 1, noi
      read(*,*) input
      if (count == 1) then
         maximum = input
         minimum = input
      else
         if (input > maximum)  maximum = input
         if (input < minimum)  minimum = input
      end if
      sum = sum + input
   end do

   write(*,*)
   write(*,*)  'MAXIMUM = ', maximum
   write(*,*)  'MINIMUM = ', minimum
   write(*,*)  'AVERAGE = ', sum / noi
end program  minMaxArg

File: pl/hw2/pascal/hw1.pas
// author y.emre can ocak 9119001276
// platform Linux 5.4.18-1-MANJARO
// compiled with Free Pascal Compiler version 3.0.4 [2019/05/13] for x86_64

program hw2;
var
    c, num: Integer;
begin
    readln(num);
    for c := 0  to num do
    begin
        if c > 20 then
            writeln(c);
    end;
end.